spot-client/src/common/css/_theme-variables.scss

```diff
@@ -27,4 +27,6 @@
     --mic-preview-background-color: rgba(250, 250, 250, 0.38);
     --mic-preview-background-disabled-color: rgba(250, 250, 250, 1);
     --pending: #258AFF;
+
+    --secondary-white: #d1dbe8;
```

Comment:
I'm not sure we should include the color in the variable name as one day we will have custom theming/branding. There's a lot of general color palette cleanup to do before that though.